packages/components/src/select/tests/jest/Select.test.tsx

```diff
@@ -126,14 +126,14 @@ test("selecting an option close the menu", async () => {
         </Select>
     );
 
-    act(() => {
-        userEvent.click(getByTestId("select"));
+    await act(() => {
```

Comment:
How come you need to `await` an `act` ?

Did something change with the technology?

Reply:
https://github.com/testing-library/user-event/issues/1018 

https://testing-library.com/docs/user-event/convenience/#keyboard

userEvent.click now returns a promise, I have an issue where using 14.4.0 some tests are broken especially the one using userEvent.tab, adding an await fixes the issue.